python/flask/requirements.txt

```diff
@@ -0,0 +1,10 @@
+blinker==1.4
+click==6.7
+contextlib2==0.5.5
+Flask==0.12.2
+itsdangerous==0.24
```

Comment:
are these all needed? should try to have a minimal subset if possible

Reply:
Used pipdeptree to analyze Flask transitive dependencies, and all of these are default.

Reply:
Added ReadMe @MaxBittker - let me know what you think?